public/scss/includes/fonts/inter-ui.scss

```diff
@@ -0,0 +1,200 @@
+@font-face {
+  font-family: 'Inter';
+  font-style:  normal;
+  font-weight: 100;
+  font-display: swap;
+  src: url("/fonts/Inter/Inter-Thin.woff2?v=3.18") format("woff2"),
+       url("/fonts/Inter/Inter-Thin.woff?v=3.18") format("woff");
```

Comment:
Not blocking, but I wonder if we could do some Sass `@mixin` that embeds the font name/formats/version to DRY this up a wee bit.

Possibly a less sad version of:

```scss
@mixin font($family, $src, $style: "normal", $weight: 500, $display: "swap", $version: 3.19) {
  @font-face {
      font-family: "#{$family}";
      font-style:  $style;
      font-weight: $weight;
      font-display: $display;
      src: url("#{$src}.woff2?v=#{$version}") format("woff2"),
        url("#{$src}.woff?v=#{$version}") format("woff");
  }
}

@include font(Inter, /fonts/Inter/Inter-Thin, normal, 100);
@include font(Inter, /fonts/Inter/Inter-ThinItalic, italic, 100);
@include font(Inter, /fonts/Inter/Inter-ExtraLight, normal, 200);
@include font(Inter, /fonts/Inter/Inter-ExtraLightItalic, italic, 200);
@include font(Inter, /fonts/Inter/Inter-Light, normal, 300);
@include font(Inter, /fonts/Inter/Inter-LightItalic, italic, 300);
@include font(Inter, /fonts/Inter/Inter-Regular, normal, 400);
@include font(Inter, /fonts/Inter/Inter-Italic, italic, 400);
@include font(Inter, /fonts/Inter/Inter-Medium, normal, 500);
@include font(Inter, /fonts/Inter/Inter-MediumItalic, italic, 500);
@include font(Inter, /fonts/Inter/Inter-SemiBold, normal, 600);
@include font(Inter, /fonts/Inter/Inter-SemiBoldItalic, italic, 600);
@include font(Inter, /fonts/Inter/Inter-Bold, normal, 700);
@include font(Inter, /fonts/Inter/Inter-BoldItalic, italic, 700);
@include font(Inter, /fonts/Inter/Inter-ExtraBold, normal, 800);
@include font(Inter, /fonts/Inter/Inter-ExtraBoldItalic, italic, 800);
@include font(Inter, /fonts/Inter/Inter-Black, normal, 900);
@include font(Inter, /fonts/Inter/InterBlackItalic, italic, 900);
```

It might reduce the boilerplate from like 130 lines of code down to ~30, and make it easier to manage version numbers (just change the default asset `version` number in the `@mixin`).

Reply:
It's a bit of a weak and inflexible `@mixin` (doesn't allow for `@font-face` declarations that _only_ specify .woff2 assets, or optional `local()` sources like the Metropolis use case.

Also, I guess if you reversed `$weight:500` and `$style:normal` args, you could shave a few more chars since the default `$style` is "normal". Or even merge the `normal` vs `italic` style declarations, or use a loop for normal-vs-italic, or...:

```scss
@mixin font($family, $src, $weight: 500, $display: swap, $version:3.19) {
  $styleMap: (normal: "", italic: "Italic");
  @each $name, $style in $styleMap {
    @font-face {
      font-family: "#{$family}";
      font-style:  $name;
      font-weight: $weight;
      font-display: $display;
      src: url("#{$src}#{$style}.woff2?v=#{$version}") format("woff2"),
        url("#{$src}#{$style}.woff?v=#{$version}") format("woff");
    }
  }
}

@include font(Inter, /fonts/Inter/Inter-Thin, 100);
@include font(Inter, /fonts/Inter/Inter-ExtraLight, 200);
@include font(Inter, /fonts/Inter/Inter-Light, 300);
@include font(Inter, /fonts/Inter/Inter-Regular, 400);
@include font(Inter, /fonts/Inter/Inter-Medium, 500);
@include font(Inter, /fonts/Inter/Inter-SemiBold, 600);
@include font(Inter, /fonts/Inter/Inter-Bold, 700);
@include font(Inter, /fonts/Inter/Inter-ExtraBold, 800);
@include font(Inter, /fonts/Inter/Inter-Black, 900);
```

### OUTPUT

```css
@font-face {
  font-family: "Inter";
  font-style: normal;
  font-weight: 100;
  font-display: swap;
  src: url("/fonts/Inter/Inter-Thin.woff2?v=3.18") format("woff2"), url("/fonts/Inter/Inter-Thin.woff?v=3.18") format("woff");
}
@font-face {
  font-family: "Inter";
  font-style: italic;
  font-weight: 100;
  font-display: swap;
  src: url("/fonts/Inter/Inter-ThinItalic.woff2?v=3.18") format("woff2"), url("/fonts/Inter/Inter-ThinItalic.woff?v=3.18") format("woff");
}

@font-face {
  font-family: "Inter";
  font-style: normal;
  font-weight: 200;
  font-display: swap;
  src: url("/fonts/Inter/Inter-ExtraLight.woff2?v=3.18") format("woff2"), url("/fonts/Inter/Inter-ExtraLight.woff?v=3.18") format("woff");
}
@font-face {
  font-family: "Inter";
  font-style: italic;
  font-weight: 200;
  font-display: swap;
  src: url("/fonts/Inter/Inter-ExtraLightItalic.woff2?v=3.18") format("woff2"), url("/fonts/Inter/Inter-ExtraLightItalic.woff?v=3.18") format("woff");
}
…
```

Or another map for the weights...

```scss
$weightMap: (
    Thin: 100,
    ExtraLight: 200,
    Light: 300,
    Regular: 400,
    Medium: 500,
    SemiBold: 600,
    Bold: 700,
    ExtraBold: 800,
    Black: 900
);
@each $name, $weight in $weightMap {
  @include font(Inter, "/fonts/Inter/Inter-#{$name}", $weight)
}
```

Reply:
Note – I moved the font directory size issue to https://github.com/mozilla/blurts-server/issues/2223

Also, I'm not sweating this too much as it gets compiled and minified. 

**The larger question is…Do we need all these different font weights?!**

Reply:
> The larger question is…Do we need all these different font weights?!

Do we need 9 different font weights, each with 2 variations (normal-vs-italics)? I'd strongly argue "no", but I'm also guessing/hoping the vast majority of these are unused (but I'll verify later in production).

I quickly poked at https://monitor-v2.herokuapp.com/breaches and looked at DevTools > Network and it looks like it is only loading 3 fonts currently (which is around 60 KB of font assets):
- https://monitor-v2.herokuapp.com/fonts/Metropolis/Metropolis-Bold.woff2
- https://monitor-v2.herokuapp.com/fonts/Metropolis/Metropolis-SemiBold.woff2
- https://monitor-v2.herokuapp.com/fonts/Metropolis/Metropolis-Medium.woff2